pages/index.js

```diff
@@ -63,15 +63,25 @@ export default function HomePage({ bookmarkedRecipesIDs, onToggleBookmark }) {
   const currentTip = recipes[currentTipIndex];
   return (
     <div>
-      {isPopupOpen ? <Backdrop onClick={closePopup} /> : null}
+      {isPopupOpen ? (
+        <Backdrop
+          onClick={closePopup}
+          role="button"
+          aria-label="Close tip popup"
+          tabIndex="0"
+        />
```

Comment:
Wait, isnt that the background? Kinda funny to have a label for that :D
If i am confusing the "backdrop" with something else forget about it, but if its the case then its like labeling the html tag of the page.
A blind person doesn't know where the popup ends and the background is not selectable by keyboard i would say.